test/issue752_test.dart

```
@@ -0,0 +1,19 @@
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('ProductPackaging bug', () {
+    final testProduct = Product();
+    const productName = 'Orange';
+    testProduct.productName = productName;
+    testProduct.packagings = [ProductPackaging()..quantityPerUnit = '75cl'];
+    final productJson = testProduct.toJson();
+    try {
```

Would be easier to let the test fail naturally with an exception, but why not.